packages/bandwidth-test/Readme.md

```diff
@@ -0,0 +1,7 @@
+## bandwidth-test
```

Comment:
We could point out that running `bandwidth-test -h` (actually also `bandwidth-test hakuna matata` gives the same output, but `-h` looks more professional) outputs an usage help.
Or we could just copy and paste here the help text which can be taken from [here](https://github.com/libremesh/lime-packages/blob/master/packages/bandwidth-test/files/bin/bandwidth-test.lua#L47), what do you prefer?

Reply:
In my case "bandwidth-test -h" triggers the measurements, it doesn't show the help.

`root@ql-anaymarcos:~# bandwidth-test -h
Attempting connection to http://speedtest-lon1.digitalocean.com/10mb.test
`

Reply:
LOL sorry, I'm going to fix that XD